### ISSUE TITLE
Disabled buttons shouldn't be clickable

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1629]
In the new document modal (and dropdown) within CryptDrive, the ‘Document’ and ‘Presentation’ options appear visually disabled but are still clickable, which is an inconsistency that may confuse users. Additionally, the visually disabled status of these options isn't announced by screen readers. To resolve this, the buttons should either be fully disabled to prevent interaction or be styled to appear active, guiding users to a page that clearly informs them that the feature is currently unavailable, as is done now.
This issue also occurs in the ‘filter’ dropdown menu; here, the ‘Document’ and ‘Presentation’ options should be either fully active or fully disabled.
![image](https://github.com/user-attachments/assets/71c3496d-1421-4259-acc8-cc57f0ff6454)
